### DISCUSSION TITLE
Is there a way to query a table and dump the results directly to a file efficiently?

### DISCUSSION THREAD

**Discussion by AdamJSoftware** [SeaQL/sea-orm#1806] [Q&A]
I am specifically looking at this command in sqlx [copy_out_raw](https://docs.rs/sqlx/latest/sqlx/struct.PgConnection.html#method.copy_out_raw) and wondering if this has been reimplemented in seaorm or if there are any alternatives?

**Comment by tyt2y3** [comment]
Not directly, but you can build a query into SQL statement, then call the respective SQLx method.
